### ISSUE TITLE
PaddleSpeech如何关闭log信息输出

### ISSUE THREAD

**Issue by 503718696** [PaddlePaddle/PaddleSpeech#2750]
import paddle
from paddlespeech.cli.asr.infer import ASRExecutor
asr_executor = ASRExecutor()
text = asr_executor(audio_file=audio_path,device=paddle.get_device(), force_yes=True)

上面函数方式调用，产生大量log信息日志，如何关闭？下面是日志样例。

日志样例：
[2022-12-17 00:08:09,180] [ WARNING] - The sample rate of the input file is not 16000.
                             The program will resample the wav file to 16000.
                             If the result does not meet your expectations，
                             Please input the 16k 16 bit 1 channel wav file.                         
2022-12-17 00:08:12.485 | INFO     | paddlespeech.s2t.frontend.featurizer.text_featurizer:_load_vocabulary_from_file:229 - BLANK id: 0
2022-12-17 00:08:12.486 | INFO     | paddlespeech.s2t.frontend.featurizer.text_featurizer:_load_vocabulary_from_file:230 - UNK id: 1
2022-12-17 00:08:12.486 | INFO     | paddlespeech.s2t.frontend.featurizer.text_featurizer:_load_vocabulary_from_file:231 - EOS id: 5536
2022-12-17 00:08:12.487 | INFO     | paddlespeech.s2t.frontend.featurizer.text_featurizer:_load_vocabulary_from_file:232 - SOS id: 5536
2022-12-17 00:08:12.487 | INFO     | paddlespeech.s2t.frontend.featurizer.text_featurizer:_load_vocabulary_from_file:233 - SPACE id: -1
2022-12-17 00:08:12.488 | INFO     | paddlespeech.s2t.frontend.featurizer.text_featurizer:_load_vocabulary_from_file:234 - MASKCTC id: -1
2022-12-17 00:08:12.488 | INFO     | paddlespeech.s2t.models.u2.u2:_init_from_config:829 - U2 Encoder type: conformer
2022-12-17 00:08:12.639 | INFO     | paddlespeech.s2t.modules.loss:__init__:40 - CTCLoss Loss reduction: sum, div-bs: True
2022-12-17 00:08:12.640 | INFO     | paddlespeech.s2t.modules.loss:__init__:42 - CTCLoss Grad Norm Type: None
2022-12-17 00:08:12.640 | INFO     | paddlespeech.s2t.modules.loss:__init__:73 - CTCLoss() kwargs:{'norm_by_times': False}, not support: {'norm_by_batchsize': False, 'norm_by_total_logits_len': False}
2022-12-17 00:08:14.397 | INFO     | paddlespeech.s2t.utils.tensor_utils:pad_sequence:96 - length 1, out_tensor [10, 2], tensor [1]
2022-12-17 00:08:14.397 | INFO     | paddlespeech.s2t.utils.tensor_utils:pad_sequence:96 - length 2, out_tensor [10, 2], tensor [2]
2022-12-17 00:08:14.399 | INFO     | paddlespeech.s2t.utils.tensor_utils:pad_sequence:96 - length 1, out_tensor [10, 2], tensor [1]
2022-12-17 00:08:14.399 | INFO     | paddlespeech.s2t.utils.tensor_utils:pad_sequence:96 - length 2, out_tensor [10, 2], tensor [2]
2022-12-17 00:08:14.400 | INFO     | paddlespeech.s2t.utils.tensor_utils:pad_sequence:96 - length 2, out_tensor [10, 2], tensor [2]
2022-12-17 00:08:14.401 | INFO     | paddlespeech.s2t.utils.tensor_utils:pad_sequence:96 - length 2, out_tensor [10, 2], tensor [2]
2022-12-17 00:08:14.401 | INFO     | paddlespeech.s2t.utils.tensor_utils:pad_sequence:96 - length 1, out_tensor [10, 2], tensor [1]
2022-12-17 00:08:14.402 | INFO     | paddlespeech.s2t.utils.tensor_utils:pad_sequence:96 - length 2, out_tensor [10, 2], tensor [2]
2022-12-17 00:08:14.402 | INFO     | paddlespeech.s2t.utils.tensor_utils:pad_sequence:96 - length 2, out_tensor [10, 2], tensor [2]
2022-12-17 00:08:14.403 | INFO     | paddlespeech.s2t.utils.tensor_utils:pad_sequence:96 - length 1, out_tensor [10, 2], tensor [1]

**Comment by yt605155624** [comment]
你用的不是最新版吧，可以安装最新版试试。
或者在这里改下日志级别 https://github.com/PaddlePaddle/PaddleSpeech/blob/979bbd9dcb6b2214692f4751d2e5cf8dc476c806/paddlespeech/cli/log.py#L52
参考 https://github.com/PaddlePaddle/PaddleSpeech/issues/2387

**Comment by 503718696** [comment]
修改成：self.logger.setLevel(logging.ERROR)
这样就好了，谢谢
